generation/Windows/um/coml2api/generate.rsp

```diff
@@ -0,0 +1,16 @@
+@../../../settings.rsp
+@../../../remap.rsp
+--file
+um-coml2api.h
+--methodClassName
+Windows
+--namespace
+TerraFX.Interop.Windows
+--output
+../../../../sources/Interop/Windows/Windows/um/coml2api
+--test-output
+../../../../tests/Interop/Windows/Windows/um/coml2api
+--traverse
+C:/Program Files (x86)/Windows Kits/10/Include/10.0.20348.0/um/coml2api.h
+--with-librarypath
+*=ole32
```

Comment:
nit: Can we replace this with:
```suggestion
FmtIdToPropStgName=ole32
GetConvertStg=ole32
GetHGlobalFromILockBytes=ole32
PropStgNameToFmtId=ole32
ReadClassStg=ole32
ReadClassStm=ole32
StgCreateDocfile=ole32
StgCreateDocfileOnILockBytes=ole32
StgCreatePropSetStg=ole32
StgCreatePropStg=ole32
StgCreateStorageEx=ole32
StgIsStorageFile=ole32
StgIsStorageILockBytes=ole32
StgOpenPropStg=ole32
StgOpenStorage=ole32
StgOpenStorageEx=ole32
StgOpenStorageOnILockBytes=ole32
StgSetTimes=ole32
WriteClassStg=ole32
WriteClassStm=ole32
```

While ClangSharp supports `*=...` here, I've been switching to explicitly listing APIs instead to help catch things which are "undocumented" and which shouldn't have corresponding bindings.